### PR TITLE
Add a JBossEAP on RHEL image

### PR DESCRIPTION
**Pull request by pmorie**

### CODE REVIEW

File: rhel-jbosseap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+FROM 10.64.27.125:5000/vpavlin/rhel6:latest
```
Comment (smarterclayton) [inline rhel-jbosseap/Dockerfile]: Make this rhel6:latest so someone can import locally and run

File: rhel-jbosseap/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# Install mysql module
+ADD mysql-module.xml /opt/jboss-eap-6.2/modules/system/layers/base/com/mysql/jdbc/main/module.xml
+ADD mysql-connector-java-5.1.30-bin.jar /opt/jboss-eap-6.2/modules/system/layers/base/com/mysql/jdbc/main/mysql-connector-java.jar
+
+# Install the mongodb module
+ADD mongodb-module.xml /opt/jboss-eap-6.2/modules/system/layers/base/com/mongodb/main/module.xml
+ADD mongo-java-driver-2.9.3.jar /opt/jboss-eap-6.2/modules/system/layers/base/com/mongodb/main/mongo-java-driver.jar
+
+# Remove unneeded zip files
+RUN rm /opt/jboss-eap*.zip
```
Comment (smarterclayton) [inline rhel-jbosseap/Dockerfile]: Doesn't do any good....

File: rhel-jbosseap/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+# Add the launch script
+ADD launch /opt/jboss-eap-6.2/bin/
+
+# Specify default values for entry point
+CMD ["/eap/bin/launch", "standalone", "-c", "standalone-ha.xml", "-b", "0.0.0.0"]
+
+# Add scripts for sti compatibility
+ADD ./prepare /usr/bin/prepare
+ADD ./run /usr/bin/run
```
Comment (smarterclayton) [inline rhel-jbosseap/Dockerfile:43]: Run seems pointless given CMD

File: rhel-jbosseap/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+rhel-jbosseap
+=============
+
+JBoss EAP on RHEL.
```
Comment (smarterclayton) [inline rhel-jbosseap/README.md]: JBoss Enterprise Application Platform 6.x (EAP) Red Hat Enterprise Linux 6

File: rhel-jbosseap/Dockerfile (hunk 3 above)
Comment (pmorie) [inline rhel-jbosseap/Dockerfile:43]: I completely agree, but the run script is not yet optional.

**Pull request merged**